lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -76,6 +77,20 @@ define((require, exports, module) ->
                     else
                         rightNearest = rightCandidate
             key++
+
+        #Special case related to positionLeftwards has not next char and positionLeftWards.column += 1 does nothing.
+        if session.getLine(positionRightwards.row + 1) == '' 
```

Comment:
@dbarashev 
no, if we have tone of text afterwards cursor, `rightNearest == null`  false